### DISCUSSION TITLE
Parcel 2: cannot import SVGs

### DISCUSSION THREAD

**Discussion by Tassil0** [parcel-bundler/parcel#4923] [General]
Hello,
I've tried Parcel 2 because of the bug that causes css to not be loaded in sub pages. Now I can't import SVGs like I did in Parcel, is there a different way?

Code:
`import SVG from './picture.svg';`

I get this error:
```
🚨 Build failed.
Error: No transformers found for "picture.svg".
```

**Comment by mischnic** [comment]
Does this help https://v2.parceljs.org/getting-started/migration/#importing-non-code-assets-from-javascript?

**Reply by Tassil0** [reply]
Now I get this:

```
🚨 Build failed.
@parcel/packager-html: The expression evaluated to a falsy value:

  (0, assert.default)(resolved.type === 'bundlegroup')

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

  (0, _assert.default)(resolved.type === 'bundle_group')

    at replaceURLReferences (project/node_modules/@parcel/utils/lib/replaceBundleReferences.js:58:25)
    at Object.package (project/node_modules/@parcel/packager-html/lib/HTMLPackager.js:64:44)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Reply by Tassil0** [reply]
That was using `serve`

When I `build` I get this:

```
🚨 Build failed.
Error: Bundles must have unique filePaths
AssertionError [ERR_ASSERTION]: Bundles must have unique filePaths
    at BundlerRunner.nameBundles (project/node_modules/@parcel/core/lib/BundlerRunner.js:172:21)
```

**Reply by mischnic** [reply]
Are you using `yarn add parcel@nightly`?

**Reply by Tassil0** [reply]
I was using `parcel@next`, `parcel@nightly` works just fine in a test project, but the build process is somehow cycled in my project.
Guess I'll have to wait until the feature makes it into `parcel@next`. Thanks for your help!
